src/runner.js

```diff
@@ -170,7 +187,10 @@ class Runner {
       }
 
       const optionResults = new results.OptionsResult(filteredSelections.map((s) => {
-        return s.text;
+        const formater = this.messageFormater.compile(s.text);
+
+        // Return the Text Result of the option
+        return new results.TextResult(formater(this.variables.data), s, s.lineNum, formater);
```

Comment:
no need to attach formatter to the result here and with Options. Please remove the last parameter you added

Reply:
We can keep the scope of the pr small and just do 
`return formater(s.text);`

returning a TextResult here may lead to stuff breaking in existing wrappers and we are not going to gain much from the extra data attached to option result text